Gruntfile.js

```diff
@@ -59,6 +59,38 @@ module.exports = function(grunt) {
           {expand: true, cwd: 'test/fixtures/', src: ['**'], dest: './'}
         ]
       },
+      zipDoNotCreateEmptyArchiveTrue: {
+        options: {
+          doNotCreateEmptyArchive: true,
+          archive: function () {
+            return 'tmp/compress_test_files_empty_must_not_be_created_because_option_set_to_true.zip';
```

Comment:
What do you suggest? I afraid that name is too long. It is so only for one reason: name explains why this file is exists. Should I use some really short name or what? Like `zip_shouldnt_be_created_1.zip`?